mmv1/third_party/tgc/gkehub_membership.go

```diff
@@ -0,0 +1,662 @@
+package google
+
+import (
+	"reflect"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v5/tfplan2cai/converters/google/resources/cai"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google-beta/google-beta/transport"
+)
+
+const GkehubMembershipAssetType string = "gkehub.googleapis.com/Membership"
+
+func resourceGkehubMembership() cai.ResourceConverter {
+	return cai.ResourceConverter{
+		AssetType: GkehubMembershipAssetType,
+		Convert:   GetGkehubMembershipCaiObject,
+	}
+}
+
+func GetGkehubMembershipCaiObject(d tpgresource.TerraformResourceData, config *transport_tpg.Config) ([]cai.Asset, error) {
+	name, err := cai.AssetName(d, config, "//gkehub.googleapis.com/projects/{{project}}/locations/{{location}}/memberships/my-membership")
+	if err != nil {
+		return []cai.Asset{}, err
+	}
+	if obj, err := GetGkehubMembershipApiObject(d, config); err == nil {
+		return []cai.Asset{{
+			Name: name,
+			Type: GkehubMembershipAssetType,
+			Resource: &cai.AssetResource{
+				Version:              "v1",
+				DiscoveryDocumentURI: "https://gkehub.googleapis.com/$discovery/rest?version=v1",
```

Comment:
Can you verify if "version=v1" is present in this field of the CAI list asset response? Let us remove it if that is not the case.

Reply:
done, thanks.